src/main/java/hudson/plugins/active_directory/ActiveDirectoryAuthenticationProvider.java

```diff
@@ -68,6 +73,26 @@
      */
     private final _Connection con;
 
+    /**
+     * The {@link UserDetails} cache.
+     *
+     * By default set-up with size 1000 and ttl 3600 for legacy reasons
+     */
+    Cache<String, UserDetails> userCache = CacheBuilder.newBuilder()
+            .maximumSize(1000)
+            .expireAfterWrite(3600, TimeUnit.SECONDS)
+            .build();
+
```

Comment:
These default values don't match those in the help, and those ones seem more sensible.